opensir/models/model.py

```diff
@@ -175,10 +181,34 @@ def fit(self, t_obs, n_i_obs, population, fit_index=None):
             Model: Reference to self
         """
 
+        # If input not np.ndarray, cast
+        if not isinstance(t_obs, np.ndarray):
```

Comment:
as said before, these lines could be dropped if we only accept np.array

Reply:
ping @felipehuerta17 :)

Reply:
Hi @jia200x , please look at the most recent commits where an error is raised if the user doesn't provide numpy arrays

Best